core/chains/evm/client/errors.go

```diff
@@ -123,20 +126,21 @@ var arbitrumFatal = regexp.MustCompile(`(: |^)(invalid message format|forbidden
 var arbitrum = ClientErrors{
 	// TODO: Arbitrum returns this in case of low or high nonce. Update this when Arbitrum fix it
 	// https://app.shortcut.com/chainlinklabs/story/16801/add-full-support-for-incorrect-nonce-on-arbitrum
-	NonceTooLow: regexp.MustCompile(`(: |^)invalid transaction nonce$|(: |^)nonce too low(:|$)`),
-	// TODO: Is it terminally or replacement?
+	NonceTooLow:           regexp.MustCompile(`(: |^)invalid transaction nonce$|(: |^)nonce too low(:|$)`),
 	TerminallyUnderpriced: regexp.MustCompile(`(: |^)gas price too low$`),
 	InsufficientEth:       regexp.MustCompile(`(: |^)(not enough funds for gas|insufficient funds for gas \* price \+ value)`),
 	Fatal:                 arbitrumFatal,
+	L2FeeTooLow:           regexp.MustCompile(`(: |^)max fee per gas less than block base fee(:|$)`),
```

Comment:
`L2FeeTooLow` is not really new. Both because we already did a version of it for Optimism, and because we've been testing it for a few weeks now. `L2Full` is new, but IMO it is pretty straightforward :shrug: Can't really test it though.

Reply:
Yes, I meant just the L2Full. It was primarily to have their eyes on our changes, so they're aware how we use their messages.
But it is ok, your decision. We can send them an update later too, about what support we added in this release.